frontend/src/components/login/login-callback/LoginCallback.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useContext, useEffect, useState } from 'react'
+
+import { AuthContext } from 'contexts/auth.context'
+
+import { getUser, loginWithSgid, setUserAnalytics } from 'services/auth.service'
+
+import { sendException } from 'services/ga.service'
+
+const DASHBOARD_PAGE = '/'
+const LOGIN_PAGE = '/ogp-login'
```

Comment:
`nit:` This string `ogp-login` is in quite a number of places. Any opportunity to consolidate them into one constant?

Reply:
I considered creating a new folder + file `src/constants/routes.tsx` for this but I wasn't sure if it was necessary. 

Currently, the `ogp-login` string can be found in:
- `App.tsx`
- `LoginCallback.tsx`
- `LoginInput.tsx`

Once we launch the feature fully (available to all whitelisted government officers), all the routes will be changed to the default login route. Should we still try consolidate the routes temporarily?

Reply:
Nope, not about the organisation of routes but about this string literal `ogp-login`.

But noted on the route being deprecated in the near future. Now I think we don't need to persist `ogp-login` as a constant.

So, noop!